Libraries/LibWeb/Selection/Selection.cpp

```diff
@@ -653,4 +654,40 @@ void Selection::move_offset_to_previous_word(bool collapse_selection)
     }
 }
 
+void Selection::move_offset_to_next_line(bool collapse_selection)
+{
+    auto* text_node = as_if<DOM::Text>(anchor_node().ptr());
```

Comment:
I think this should be `focus_node()` in both methods; I've got some relevant changes for that in #5952